src/components/OwnAccountsList.tsx

```diff
@@ -39,7 +39,29 @@ const OwnAccountsList = ({ budget }: Props) => {
     accountApi
       .getAll(budget, { external: false, archived: Boolean(archived) })
       .then(data => {
-        setAccounts(data)
+        // Create list of all IDs
+        const ids: string = data.map((account: Account) => {
```

Comment:
That should be `string[]` (or even better `UUID[]`), but also you shouldn't even need to type this explicitly.